pkg/ingester/ingester.go

```diff
@@ -332,15 +327,10 @@ func (i *Ingester) append(ctx context.Context, sample *model.Sample) error {
 	}()
 
 	prevNumChunks := len(series.chunkDescs)
-	if err := series.add(model.SamplePair{
+	if err := series.add(ctx, model.SamplePair{
 		Value:     sample.Value,
 		Timestamp: sample.Timestamp,
 	}); err != nil {
-		if mse, ok := err.(*memorySeriesError); ok {
```

Comment:
I wonder if `err.(*memorySeriesError)` is the part that is wrong, extraneous `*` maybe? The path below was intended to stop `memorySeriesError` leaking back out.

Reply:
Hmm... it looks correct to me.  The issue cleared itself up after the deployment finished so I wonder if I barking up completely the wrong tree?

Reply:
Yeah I'm completely wrong on the cause of this, will do some more digging.